### ISSUE TITLE
terminado seems to ignore shell

### ISSUE THREAD

**Issue by dsblank** [jupyter/jupyterhub#133]
Moving item from terminado to here:

https://github.com/takluyver/terminado/issues/2

See issue for details.

**Comment by minrk** [comment]
Thanks

**Comment by dsblank** [comment]
Did you mean to close? If so, what is resolution... user error? :) Could well be.

**Comment by minrk** [comment]
Arg. Read the description backwards, I thought you were moving from here to there.

**Comment by minrk** [comment]
@dsblank you are using the sudospawner, right?

**Comment by dsblank** [comment]
Yes:
```
jupyterhub --JupyterHub.spawner=sudospawner.SudoSpawner --JupyterHub.ip=165.106.10.83 --JupyterHub.port=80  --JupyterHub.data_files_path='/home/dblank/jupyter.brynmawr'
```

**Comment by jdavidheiser** [comment]
I made a simple change to spawner.py that seems to fix this, and pass on the SHELL environment variable.  Adding an export SHELL=bash before launching jupyterhub now works to give us bash in the terminado shells launched by our jupyter sessions.

**Comment by minrk** [comment]
closed by #204
